<launch>

    <!-- Input arguments -->
    <arg name="input_image_topic" default="camera/image_raw" />
    <arg name="output_image_topic" default="filtered_image/black_and_white_image" />

    <node pkg="ros_cv_tools" type="filter_grayscale_binary" name="filter_grayscale_binary">
        <param name="input_image_topic" value="$(arg input_image_topic)" />
        <param name="output_image_topic" value="$(arg output_image_topic)" />
    </node>

</launch>